python/tk_framework_desktopserver/server_protocol.py

```diff
@@ -77,13 +87,10 @@ def connectionLost(self, reason):
 
             if certificate_error:
                 logger.info("Certificate error!")
-                StatusServerProtocol.serverStatus = StatusServerProtocol.SSL_CERTIFICATE_INVALID
```

Comment:
This code is dead. We removed the status server protocol before the initial release of the websocket server.